cmsBuild

```diff
@@ -879,12 +881,62 @@ if [ X"$(id -u)" = X0 ]; then
     exit 1
 fi
 """
-
 DEFAULT_POST_PREAMBLE = """
 if [ "X$CMS_INSTALL_PREFIX" = "X" ] ; then CMS_INSTALL_PREFIX=$RPM_INSTALL_PREFIX; export CMS_INSTALL_PREFIX; fi
 %{relocateConfig}etc/profile.d/init.sh
 %{relocateConfig}etc/profile.d/init.csh
 """
+
+
+if sys.platform == "darwin": DEFAULT_POST_POSTAMBLE = """
+case %{cmsplatf} in
```

Comment:
Yes. I thought about completely removing the case statement I will try that next.